gulpfile.mjs

```diff
@@ -378,8 +378,7 @@ function createWebpackConfig(
   const alias = createWebpackAlias(bundleDefines);
   const experiments = isModule ? { outputModule: true } : undefined;
 
-  // Required to expose e.g., the `window` object.
-  output.globalObject = "globalThis";
+
```

Comment:
If we do this, please reduce the number of newlines between the two sections here to just one instead of the current three.